RemoveOutroMod/RemoveOutroMod.lua

```diff
@@ -0,0 +1,10 @@
+ModUtil.RegisterMod("RemoveOutroMod")
```

Comment:
Just as a general naming convention question, do we want to always have the "Mod" suffix? I've been moving away from it on this project, but don't feel strongly either way

Reply:
I agree - was going to add a toggle config for this, then started working on other stuff. Will add that to the branch

Reply:
I have https://github.com/parasHadesMods/ModConfigMenu which we could use unless there are objections, should make it easy to add whichever toggles we want.

Reply:
100% will rework this to use ModConfigMenu